emscripten.py

```diff
@@ -777,6 +777,9 @@ def fix_import_name(g):
 def make_export_wrappers(exports, delay_assignment):
   wrappers = []
   for name in exports:
+    # Tags cannot be wrapped in createExportWrapper
+    if name == '__cpp_exception':
+      continue
```

Comment:
I'd like to find a better way to deal with this... it should be that we only generate wrappers for functions..  and in fact the "exports" has traditionally be reserved for just functions, with globals getting their own list.  

I'm somewhat loath to change the json format emitted by wasm-emscripten-finalize at the moment though since I am in the middle to trying to remove it in favor of just using python to extract the metadata.

Reply:
Then how about adding a TODO here to fix it properly after you fix the metadata?

Reply:
Sure, but even landed that binaryen change makes me a little sad.. seems somewhat inconsistent.   Maybe I just need to get over.